newsletters/20190807-ingress.md

```diff
@@ -0,0 +1,86 @@
+Today we talk about the Kubernetes Ingress — the component in charge of routing the traffic to your apps.
+
+Here are a few interesting facts about the Ingress.
+
+## The most misunderstood components of the cluster.
+
+In the past, we noticed that teams were rolling out their nginx reverse proxy to route traffic to the cluster.
+
+In other words, they were rewriting the Ingress controller from scratch
+
+## It's a pain to debug.
+
+Some Ingress controllers direct traffic to Services (aws-alb-ingress-controller) while others (most) skip the Service and distribute the traffic directly to Pods.
+
+How do you know if the traffic is degraded due to a slow kube-proxy or the app?
```

Comment:
If I understood the meaning of the sentence correctly, then something like this would be clearer to me:

`How can you know whether the traffic is degraded due to a slow kube-proxy or due to the app itself?`